PSC-ONBOARDING-OFFBOARDING.md

```diff
@@ -0,0 +1,163 @@
+existing `product-security-committee` field, according to the usernames
+alphabetical placing:
+
+```yaml
+aliases:
+  product-security-committee:
+    - jdoe
+```
+
+#### kubernetes/community repository
+
+##### file: https://github.com/kubernetes/community/blob/master/sigs.yaml
+
+Add / remove the PSC member(s)  github name from `community/sigs.yaml` to the
+existing `label`| `leadership` field, according to the usernames alphabetical
+placing:
+
+```yaml
+label: product-security
+  leadership:
+    chairs:
+    - github: jdoe
+      name: Jane Doe
+      company: Acme LTD
+```
+
+Once the yaml is updated, run `make generate` to regenerate the markdown
+versions.
+
+This will then automatically create the following files:
+
+* kubernetes/community/OWNERS_ALIASES
+* kubernetes/community/committee-product-security/README.md
+* kubernetes/community/sig-list.md
+
+####  Add/remove from Hackerone
+
+<Section TBD>
+
+
+####  Add/remove from OpsGenie rotation
+
+##### url: https://app.opsgenie.com/teams/dashboard/633ac733-665f-4d62-871e-cbc011f37037
+
+Once the user account has been created in OpsGenie, they must add a
+`github=GITHUB_USERNAME` tag to their OpsGenie profile for the tool to work.
+
+See the below example.
+
+![User Configuration](/images/user-tag.png)
+
+####  Add/ Remove PSC Owner permission from Kubernetes Security Mailing Lists
+
+New PSC members are added to various security lists and upgraded to "owner" (or
+downgraded to member if they are leaving the PSC)
+
+To add members and assign the "owner" role:
+
+1. Visit the groups URL as a user authenticated as an existing "owner".
+2. Select the 'Manage group' or 'Manager Members' link.
+3. Within the left side panel, select 'Direct add member'.
+4. Enter the new PSC members email address
+5. Select the 'All members' link, or search for the new member in the search box.
+6. Select the new member and change the drop down role to "Owner"
+
+---
+
+**Existing Accounts**
+
+It may be the case, that the user has already joined the group / mailing list.
+If this is the case, steps 1 to 4 can be ignored and you can proceed to change
+the role of their existing account.
+
+---
+
+To downgrade existing owners to members:
+
+1. Visit the groups URL as a user authenticated as an existing "owner".
+2. Select the 'All members' link, or search for the existing owner in the search box.
+3. Select the owner and change the drop down role to "Member".
+
+| Mailing List | URL|
+| ------------- | ------------- |
+| Security | https://groups.google.com/a/kubernetes.io/forum/#!forum/security |
+| Security Discuss | https://groups.google.com/forum/#!forum/kubernetes-security-discuss |
+| Security Discuss (Private) | https://groups.google.com/a/kubernetes.io/forum/#!forum/security-discuss-private |
+| Security Announce | https://groups.google.com/forum/#!forum/kubernetes-security-announce |
+| Distributors Announce | https://groups.google.com/a/kubernetes.io/forum/#!forum/distributors-announce |
```

Comment:
I don't see a corresponding list in the onboarding. The onboarding list should also include publish permissions on

```
| Kubernetes Announce | https://groups.google.com/forum/#!forum/kubernetes-announce |
| Kubernetes Dev | https://groups.google.com/forum/#!topic/kubernetes-dev |
```

And a verified account on discuss.kubernetes.io. My account and post got flagged initially on discuss because it was my first post

Reply:
hi @micahhausler , please check review 4a766808a1de2af2a10ad167c467e2e5e981777f